src/brouwers/shop/api/serializers.py

```diff
@@ -0,0 +1,83 @@
+from collections import OrderedDict
+from django.db.models import Q
+
+from rest_framework import serializers
+
+from brouwers.users.api.serializers import UserSerializer
+
+from ..models import Cart, CartProduct, Product
+
+
+class ProductSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Product
+        fields = ('id', 'name', 'brand', 'image', 'price', 'vat', 'categories', 'model_name')
+
+
+class ProductField(serializers.PrimaryKeyRelatedField):
+    def to_representation(self, value):
+        pk = super(ProductField, self).to_representation(value)
+        try:
+            item = Product.objects.get(pk=pk)
```

Comment:
possible performance problem here in list representations, since you can't prefetch/select_related anything?